core-tests/src/test/java/org/mule/runtime/core/DefaultMessageContextTestCase.java

```diff
@@ -13,11 +13,9 @@
 import org.mule.runtime.core.api.EventContext;
 import org.mule.runtime.core.api.MuleContext;
 import org.mule.runtime.core.api.Event;
-import org.mule.runtime.core.api.InternalMessage;
+import org.mule.runtime.core.api.message.InternalMessage;
```

Comment:
This may be done in a separate issue, but InternalMessage shouldn't be in an internal package or at least out of core.api?